### DISCUSSION TITLE
PCL动画的神奇帧率

### DISCUSSION THREAD

**Discussion by kong-que-dnf123** [Meloong-Git/PCL#3751] [讨论]
PCL的每一个动画似乎都是及其的高帧
刚才我开一个爆肝内存的整合包
我动态壁纸卡了 输入法卡了 快捷键卡了 甚至QQ音乐播放的都卡了
但我惊奇的发现PCL那个代表正在开启游戏的小稿子砸的那是一点都没有卡的意思啊
[动态表情-歪瑞谷得]

**Comment by kong-que-dnf123** [comment]
后续——MC打开了 电脑的MC窗口在苦苦挣扎后自尽了（）
![image](https://github.com/Hex-Dragon/PCL2/assets/120994373/1ab6d4cc-b5d3-4dc9-9a40-0e47a5fcf056)

**Comment by MoYuan-CN** [comment]
60帧来着（
